packages/ubuntu_desktop_installer/lib/pages/write_changes_to_disk/write_changes_to_disk_page.dart

```diff
@@ -59,45 +60,65 @@ class _WriteChangesToDiskPageState extends State<WriteChangesToDiskPage> {
       ),
       header: Text(lang.writeChangesDescription),
       content: ListView(children: <Widget>[
-        Align(
-          alignment: Alignment.centerLeft,
-          child: Text(lang.writeChangesPartitionTablesHeader),
+        SizedBox(height: MediaQuery.of(context).size.height * 0.05),
+        Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Text(
+              lang.writeChangesDevicesTitle,
+              style: Theme.of(context).textTheme.titleLarge,
+            ),
+            const SizedBox(height: kContentSpacing / 2),
+            Text(lang.writeChangesPartitionTablesHeader)
+          ],
         ),
         const SizedBox(height: kContentSpacing / 2),
-        for (final disk in model.disks)
-          Column(
+        YaruBorderContainer(
```

Comment:
Could you fill the container with the surface color, please? It gives a subtle difference in the light theme.
```dart
color: Theme.of(context).colorScheme.surface
```

Reply:
:+1: